MiguelAngelVidalLiborio/Dia6-30Agosto/hackerrank/collections/HashSetExercise.java

```diff
@@ -0,0 +1,29 @@
+package collections;
+
+import java.util.Scanner;
+import java.util.HashSet;
+import java.util.Set;
+
+public class HashSetExercise {
+	public static final Scanner SCAN = new Scanner(System.in);
+
+	public static void main(String[] args) {
+		int numberOfPairs = SCAN.nextInt();
+		String[] pair_left = new String[numberOfPairs];
```

Comment:
Local variable names must be written in lowerCamelCase
https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names

Reply:
i apologize for this mistake